packages/api-generator/src/locale/en/v-btn-toggle.json

```diff
@@ -9,6 +9,11 @@
     "tile": "Removes the component's border-radius."
   },
   "events": {
-    "change": "Emitted when the input is changed by user interaction"
+    "update:modelValue": "Emitted when the input is changed by user interaction"
+  },
+  "exposed": {
+    "next": "activates the next button",
+    "prev": "activates the previous button",
+    "select": "selects a button by index, the second parameter is a boolean to indicate if the button should be selected or not"
```

Comment:
```suggestion
    "next": "Activates the next button",
    "prev": "Activates the previous button",
    "select": "Selects a button by index, the second parameter is a boolean to indicate if the button should be selected or not"
```

Reply:
"Selects a button by index" doesn't seem to be correct though, it's actually by uid which doesn't really seem useful